ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -7,13 +7,15 @@
 	objects = {
```

Comment:
I'm assuming all file up to here are for ios development?

Reply:
Yes, I cannot explain what this one does tho LOL